var/spack/repos/builtin/packages/agile/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class Agile(AutotoolsPackage):
+    """AGILe is A Generator Interface Library (& executable), i.e. a uniform
+    object oriented C++ interface for a variety of Fortran-based Monte Carlo
+    event generators.."""
+
+    homepage = "https://agile.hepforge.org/"
+    url      = "http://www.hepforge.org/archive/agile/AGILe-1.4.1.tar.gz"
+
+    tags = ['hep']
+
+    version('1.5.1', sha256='e38536300060e4b845ccaaed824c7495944f9117a0d7e4ee74a18bf278e2012f')
+
+    depends_on('hepmc')
+    depends_on('boost')
+    depends_on('python')
+    depends_on('swig')
+    depends_on('py-future')
+
+    def configure_args(self):
+        options = ['--prefix=%s' % self.spec.prefix,
+                   '--with-hepmc=%s' % self.spec['hepmc'].prefix,
+                   '--with-boost=%s' % self.spec['boost'].prefix,
+                   '--disable-pyext',  # only possible with python2 (?)
+                   'CFLAGS=-g0 -O2',
+                   'CXXFLAGS=-g0 -O2',
+                   'FFLAGS=-g0 -O2']
```

Comment:
I tend to agree, but this recipe is mostly for archiving purposes, documenting how this package was built. It might be needed as a dependency for another generator, but mostly we just want to reproduce how it is currently built, and that happens to be the way. If there's another way to set this as a default but leave it configurable, I'm happy to do it, but I think it's ok in this case.

Reply:
Fine with me if you add yourself as a maintainer :slightly_smiling_face:

Reply:
Sure!